docs/README.md

```diff
@@ -0,0 +1,320 @@
+# rules_js documentation
+
+> Migrating from rules_nodejs? Start at the [Migration Guide](./migrate.md)
+
+Stuck?
+
+-   See the [Frequently asked questions](./faq.md)
+-   Ask in `#javascript` on <http://slack.bazel.build>
+-   Check for [known issues](https://github.com/aspect-build/rules_js/issues)
+-   Pay for support, provided by <https://aspect.dev>.
+
+## Installation
+
+From the release you wish to use:
+<https://github.com/aspect-build/rules_js/releases>
+copy the WORKSPACE snippet into your `WORKSPACE` file.
+
+Note that [bzlmod] is experimentally supported as well.
```

Comment:
Should we add a `snippet` to describe how to use `bzlmod` ?

Reply:
maybe link to `e2e/bzlmod` for the full example.